payloads/python/Makefile

```diff
@@ -75,10 +75,9 @@ endef
 all:
 ifneq ($(FROMSRC),yes)
 	${DOCKER_RUN_BUILD} \
-		-v ${TOP}:${TOP}:Z \
-		-v ${KM_OPT_RT}:${KM_OPT_RT}:Z \
-		-v ${KM_OPT_BIN}:${KM_OPT_BIN}:Z \
-		-v ${KM_OPT_LIB}:${KM_OPT_LIB}:Z \
+		-v ${TOP}:${TOP}:z \
+		${KM_OPT_VOLUME} \
+		-v ${KM_OPT_LIB}:${KM_OPT_LIB}:z \
```

Comment:
Not needed.  KM_OPT_VOLUME already has KM_OPT_LIB in it.

Reply:
Good catch, thanks!